### DISCUSSION TITLE
Οργάνωση σε κεφάλαια και ενότητες

### DISCUSSION THREAD

**Discussion by nkanagno** [courses-ionio/iv#27] [Q&A] [answered]
Καλησπέρα σας @epidrome, παρατήρησα ότι στο αποθετήριο [ioniodi/guide](https://github.com/ioniodi/guide), όλα τα αρχεία [ch0[1..7].txt](https://github.com/ioniodi/guide/tree/master/text), έχουν κενό περιεχόμενο, με μόνο ένα heading (πχ `#  ΤΑ ΕΠΤΑΝΗΣΑ ΚΑΙ ΤΟ ΙΟΝΙΟ ΠΑΝΕΠΙΣΤΗΜΙΟ` στο [ch01.txt](https://github.com/ioniodi/guide/blob/master/text/ch01.txt) ) εμείς θα πρέπει να προσθέσουμε κάποιο συγκεκριμένο έτοιμο κείμενο σε κάθε ch0[1..7].txt αρχείο?  γιατι δεν μπορεί να φτιαχτεί pdf χωρίς τα txt αρχεία να περιέχουν κάποιο κειμενικό περιεχόμενο και χωρίς να περιέχει, κάποιο md αρχείο για την μετατροπή σε latex μέσω lua pandoc.

**Comment by epidrome** [comment, marked as answer]
Τα «κεφάλαια» αντιστοιχούν στις ενότητες του οδηγού σπουδών
https://di.ionio.gr/gr/students/student-prospectus/

Η βασική προτεραιότητα για το 2ο παραδοτέο είναι να προσθέσουμε φίλτρα και μορφότυπα που θα τραβάνε το περιεχόμενο από τα συγγενικά αποθετήρια της ιστοσελίδας.

Για το 1ο παραδοτέο θα μπορούσαμε να περάσουμε copy-paste περιεχόμενο που είναι μοναδικό για τον οδηγό σπουδών και δεν υπάρχει στην ιστοσελίδα.

Όλα τα κεφάλαια. έχουν ήδη περιεχόμενο (τον τίτλο) οπότε με τις κατάλληλες εργασίες δέσμης (δανεισμένες από το βιβλίο του μαθήματος) μπορεί ήδη να δημιουργηθεί ένα αρχείο pdf (μόνο με τους τίτλους).
